packages/react-juce/package.json

```diff
@@ -17,7 +17,6 @@
     "@babel/cli": "^7.10.0",
     "@babel/core": "^7.11.1",
     "@babel/plugin-transform-runtime": "^7.11.0",
-    "@babel/plugin-proposal-class-properties": "^7.11.0",
```

Comment:
Seems like maybe an unrelated change? Although if our CI is passing then we don't really need this dependency, so, I'm cool with keeping the change

Reply:
removed it locally and seems fine, so gonna take it out.